logs/README

```diff
@@ -0,0 +1,3 @@
+Placeholder file to put the log directory in github.
```

Comment:
This is just to explain the reason for having empty log files (but we can leave it as it is suggested in this PR):
The logs files were created with a+rw permissions to facilitate development and demonstration. This setup allows internal Docker users with specific UIDs and GIDs to use the files, while also enabling different users on the host to clean the files if necessary. However, for production environments, it's better to implement stricter file permissions as you did there.

Reply:
Git does not preserve file permissions except for the execute flag, so in the original I had to change either the ownership or the file permissions of the empty log file anyway. I originally added a line to readme to do just that if you notice that two of the containers fail without any message, but then decided to add this approach, so you can slightly more safely use git commit -a.

Unfortunately, there's no way to mark a committed file as 'ignore' (which would be the real solution here), so either approach has its drawbacks. My personal preference is this one.